src/openzaak/config/tests/bootstrap/test_load_authorizations.py

```diff
@@ -0,0 +1,324 @@
+        self.assertEqual(
+            autorisatie_on.max_vertrouwelijkheidaanduiding,
+            VertrouwelijkheidsAanduiding.openbaar,
+        )
+
+    @override_settings(
+        AUTHORIZATIONS_CONFIG_DOMAIN_MAPPING_PATH=DOMAIN_MAPPING_PATH,
+        ENVIRONMENT="production",
+    )
+    def test_configure_domain_mapping(self):
+        AuthorizationConfigurationStep().configure()
+
+        self.assertEqual(JWTSecret.objects.count(), 2)
+        self.assertEqual(Applicatie.objects.count(), 2)
+        self.assertEqual(Autorisatie.objects.count(), 2)
+        self.assertEqual(CatalogusAutorisatie.objects.count(), 1)
+
+        jwt_secret_oz = JWTSecret.objects.get(identifier="open-zaak")
+        self.assertEqual(jwt_secret_oz.secret, "oz-secret")
+
+        jwt_secret_on = JWTSecret.objects.get(identifier="open-notificaties")
+        self.assertEqual(jwt_secret_on.secret, "on-secret")
+
+        applicatie_oz = Applicatie.objects.get(client_ids=["open-zaak"])
+
+        autorisatie_oz = applicatie_oz.autorisaties.get()
+        self.assertEqual(autorisatie_oz.component, ComponentTypes.zrc)
+        self.assertEqual(
+            autorisatie_oz.scopes,
+            [str(SCOPE_ZAKEN_CREATE), str(SCOPE_ZAKEN_ALLES_LEZEN)],
+        )
+        self.assertEqual(autorisatie_oz.zaaktype, "https://openzaak.nl/zaaktypen/1")
+        self.assertEqual(
+            autorisatie_oz.max_vertrouwelijkheidaanduiding,
+            VertrouwelijkheidsAanduiding.geheim,
+        )
+
+        catalogus_autorisatie_oz = applicatie_oz.catalogusautorisatie_set.get()
+        self.assertEqual(catalogus_autorisatie_oz.catalogus, self.catalogus)
+        self.assertEqual(catalogus_autorisatie_oz.component, ComponentTypes.drc)
+        self.assertEqual(
+            catalogus_autorisatie_oz.scopes, [str(SCOPE_DOCUMENTEN_ALLES_LEZEN)]
+        )
+        self.assertEqual(
+            catalogus_autorisatie_oz.max_vertrouwelijkheidaanduiding,
+            VertrouwelijkheidsAanduiding.beperkt_openbaar,
+        )
+
+        applicatie_on = Applicatie.objects.get(client_ids=["open-notificaties"])
+
+        autorisatie_on = applicatie_on.autorisaties.get()
+        self.assertEqual(autorisatie_on.component, ComponentTypes.zrc)
+        self.assertEqual(autorisatie_on.scopes, [str(SCOPE_ZAKEN_ALLES_LEZEN)])
+        self.assertEqual(
+            autorisatie_on.zaaktype, "https://external.openzaak.nl/zaaktypen/2"
+        )
+        self.assertEqual(
+            autorisatie_on.max_vertrouwelijkheidaanduiding,
+            VertrouwelijkheidsAanduiding.openbaar,
+        )
+
+    def test_configure_catalogus_does_not_exist(self):
+        """
+        Running `.configure` while some of the expected data (Catalogus) does not exist
+        should raise errors
+        """
+        self.catalogus.delete()
+
+        # Attempt to run the import
+        with self.assertRaises(ConfigurationRunFailed):
+            AuthorizationConfigurationStep().configure()
```

Comment:
I'm not sure what else we can do in case the Catalogus does not exist, not sure if we can avoid errors from being raised because we use `loaddata`

Reply:
On second thought, it currently will only be possible to run this step after catalogi have been configured (be it manually or via some fixture or imports), because otherwise this step will fail

Reply:
I think raising `ConfigurationRunFailed` if the auth fixture is not correct is an expected behaviour :+1:

Reply:
I think so too, and with the change to `bin/setup_configuration.sh` to load fixtures first if `AUTHORIZATIONS_CONFIG_ENABLE` is True, this is no longer an issue